src/client/components/NavBar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "../App.css";
+import { Link } from "react-router-dom";
+import logo from "../assets/images/food.png";

```

Comment:
you did it right i can see the img there 
sorry if I confused you 
there Is miss understanding https://github.com/JiuTak/meal-sharing/pull/2/files/ab9f936619ae1e9ff37c32602de7b93cdd9b34d2#r765503011
![jiu](https://user-images.githubusercontent.com/75618135/145770720-4824a155-9d4d-45e3-978b-0a36c0c1a1ff.png)